src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.figures;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int upperBase;
+    private int lowerBase;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, String name, int upperBase, int lowerBase, int height) {
+        super(color, name = "Isosceles trapezoid");
```

Comment:
In the constructor, the assignment `name = "Isosceles trapezoid"` is unnecessary since `name` is being set through the `super` call. You should remove `name = "Isosceles trapezoid"` and just call `super(color, "Isosceles trapezoid")`.